vcnc-rest/static/Makefile.am

```diff
@@ -12,7 +12,7 @@
 all:
 	if [ ! -d $(top_srcdir)/static/swagger-ui ]; then \
 	  (cd $(top_srcdir)/static; \
-	  $(PROG_GIT) clone https://github.com/swagger-api/swagger-ui.git; \
+	  $(PROG_GIT) clone --depth 1 --branch v2.2.10 https://github.com/swagger-api/swagger-ui.git; \
```

Comment:
I don't see any reason to.  This is much simpler (it's already done!) and we are guaranteed to get exactly the same thing every time.  Third-party is most useful when a _build_ needs to be done.